### PR TITLE
Condense config structs

### PR DESCRIPTION
**Pull request by oxade**
Condensed AuthorityPrivateInfo with AuthorityInfo since the former no longer contains private info

### CODE REVIEW

**Comment by oxade** [comment]
@velvia @longbowlu @patrickkuo 
Are there any implications w.r.t changing the deployment configs?

File: sui/src/config/mod.rs [hunk 1]
```diff
@@ -61,16 +61,8 @@ pub const AUTHORITIES_DB_NAME: &str = "authorities_db";
...
-pub struct AuthorityInfo {
-    #[serde(serialize_with = "bytes_as_hex", deserialize_with = "bytes_from_hex")]
-    pub name: AuthorityName,
-    pub host: String,
-    pub base_port: u16,
-}
-
 #[derive(Serialize, Debug, Clone)]
-pub struct AuthorityPrivateInfo {
+pub struct AuthorityInfo {
```
Comment (longbowlu) [inline sui/src/config/mod.rs:65]: don't we need `derive(Deserialize)`?

**Review by longbowlu — APPROVED** [review]
thanks for consolidating these! I don't think we need to change genesis config for this change

**Review by velvia — APPROVED** [review]
Thanks @oxade

**Pull request merged**